tracks/dart/exercises/two-fer/mentoring.md

```diff
@@ -0,0 +1,36 @@
+### Reasonable solutions
+
+The most simple solution is to set the default value for the parameter to `"you"` and use string interpolation:
+
+```dart
+String twofer([String name = "you"]) {
+  return "One for $name, one for me.";
+}
+```
+
+Solution using an expression-bodied method:
+
+```dart
+String twofer([String name = "you"]) => "One for $name, one for me.";
+```
+
+### Common suggestions
+
+- If they are using overloaded methods, suggest to use a single method with an 
+[optional parameter][reference-optional-parameter].
+- Instead of using an if-statement, suggest to use a single expression. This will force the student 
+to remove any duplication in the code.
+- Suggest using [string interpolation][reference-string-interpolation] over string concatenation.
+- If they are using `null` as the default value, suggest to use `"you"` instead.
+- Not everyone likes them, but it might be useful to suggest writing the `twofer` method as an 
+[expression-bodied method][reference-expression-bodied-method], as it is perfect for these kinds of 
```

Comment:
I would at least include a note that these are using the arrow syntax as discussed in the [language tour of Dart](https://dart.dev/guides/language/language-tour#functions).